server.js

```diff
@@ -25,3 +65,36 @@ serve(async (req) => {
     enableCors: true,
   });
 });
+
+//terminal実行
+/*
+const terminal_国際地点番号表 = function(){//タブ区切りのtxtデータを{国際地点番号 : {緯度経度(10進数)}}のJsonデータに変換する
+  const 緯度経度10進 = (str)=>{
+    const axis = str.charAt(str.length-1)
+    let pm = 0;
+    switch(axis){
+      case "N":
+      case "E":
+        pm = 1;
+        break;
+      case "S":
+      case "W":
+        pm = -1;
+        break;
+    }
+    const values = str.slice(0,-1).split(" ").map(s=>parseInt(s));
+    return pm*(values[0]+(values[1]+values[2]/60)/60)
+  }
+  const txtData = await fetch("https://www.jma.go.jp/jma/kishou/books/station/station.txt")
+  const txtData2 = await txtData.text();
+  const lines = txtData2.split('\n');
+  const region2s = lines.filter(l=>l.charAt(0)=='2')
+  const region2s2 = region2s.map(d=>d.split('\t'))
+  const japans = region2s2.filter(d=>d[1]=="JAPAN ")
+  let data = {};
+  for(let ja of japans){
+    data[ja[2]] = {latitude:緯度経度10進(ja[6]),longitude:緯度経度10進(ja[7])}
+  }
+  await Deno.writeTextFile("indexNbr.json", JSON.stringify(data));
+}
+*/
```

Comment:
```suggestion

//terminal実行
/*
const terminal_国際地点番号表 = function(){//タブ区切りのtxtデータを{国際地点番号 : {緯度経度(10進数)}}のJsonデータに変換する
  const 緯度経度10進 = (str)=>{
    const axis = str.charAt(str.length-1)
    let pm = 0;
    switch(axis){
      case "N":
      case "E":
        pm = 1;
        break;
      case "S":
      case "W":
        pm = -1;
        break;
    }
    const values = str.slice(0,-1).split(" ").map(s=>parseInt(s));
    return pm*(values[0]+(values[1]+values[2]/60)/60)
  }
  const txtData = await fetch("https://www.jma.go.jp/jma/kishou/books/station/station.txt")
  const txtData2 = await txtData.text();
  const lines = txtData2.split('\n');
  const region2s = lines.filter(l=>l.charAt(0)=='2')
  const region2s2 = region2s.map(d=>d.split('\t'))
  const japans = region2s2.filter(d=>d[1]=="JAPAN ")
  let data = {};
  for(let ja of japans){
    data[ja[2]] = {latitude:緯度経度10進(ja[6]),longitude:緯度経度10進(ja[7])}
  }
  await Deno.writeTextFile("indexNbr.json", JSON.stringify(data));
}
*/
```

この部分に関しては、`indexNbr.json` の生成用のdenoスクリプトとして別ファイルに書けるといいですね。

Reply:
なぜかコミットサジェスチョンが効かなかったので直接変更しておきました